UI/forms/AutoConfigStreamPage.ui

```diff
@@ -30,7 +30,7 @@
     <number>0</number>
    </property>
    <item>
-    <widget class="QWidget" name="widget" native="true">
+    <widget class="QWidget" name="widget" >
```

Comment:
Please clean up the lingering spaces between `" >`

Reply:
Done. Thanks.